src/elements/content-sidebar/versions/VersionsSidebar.scss

```diff
@@ -37,3 +37,8 @@
         padding-bottom: 20px;
     }
 }
+
+.upgradeNowButton {
+    color: #0061d5;
+    font-weight: bold !important;
```

Comment:
We should almost always avoid using `!important`.

Reply:
So I haven't been able to get the font-weight to be bold without the !important. I suspect this is because I am using the <PlainButton> component which creates a button with the class `btn-plain` which sets `font: inherit;`. I will check with my team on how to accomplish overriding that inherit without an `!important`

Reply:
Yeah let's dig more into what CSS classes we can use to target it @jlarsen-box and we'll figure it out.

Reply:
did it!